pkgs/development/tools/documentation/doxygen/default.nix

```diff
@@ -19,16 +30,24 @@ stdenv.mkDerivation rec {
   ];
 
   buildInputs = [ libiconv ]
+    ++ lib.optionals withSqlite [ sqlite.dev ]
     ++ lib.optionals (qt5 != null) (with qt5; [ qtbase wrapQtAppsHook ])
     ++ lib.optionals stdenv.isDarwin [ CoreServices ];
 
   cmakeFlags =
-    [ "-DICONV_INCLUDE_DIR=${libiconv}/include" ] ++
-    lib.optional (qt5 != null) "-Dbuild_wizard=YES";
+    [ "-DICONV_INCLUDE_DIR=${libiconv}/include" ]
```

Comment:
```
  cmakeFlags = [ "-DICONV_INCLUDE_DIR=${libiconv}/include" ]
```

Reply:
Why?

Reply:
Adopted locally, will be fixed on next force-push.